elastic/kibana
# [Ingest Manager] Implement concurrency control for package configs

## Summary

Resolves #69992. This PR enables concurrency control on package config saved objects by:

- Surface `version` field provided by saved objects to get package config(s) requests
- Send `version` field as part of the payload for update package config requests, where the saved object client will perform version check and throw a 409 if there is a conflict
- Display a toast message in the UI for 409 responses

**Testing:**
- Open Edit integration in two tabs
- Submit the first tab, then attempt to submit the second tab
- First tab should be successful whereas the second tab should display an error toast message

![image](https://user-images.githubusercontent.com/1965714/86419987-db0fab00-bc89-11ea-89f9-dca87e7aba64.png)
